tokio/tests/rt_threaded.rs

```diff
@@ -588,6 +589,99 @@ async fn test_block_in_place4() {
     tokio::task::block_in_place(|| {});
 }
 
+#[test]
+#[cfg(not(tokio_cross))]
+fn test_tuning() {
```

Comment:
maybe i overlooked it, but...does this test ever actually assert anything? how do we know if the tuning is working correctly?

Reply:
If it doesn't work, it runs forever.